packages/web-components/fast-components-msft/README.md

```diff
@@ -0,0 +1,12 @@
+# FAST Components MSFT
+
+## Development
+
+To start the component development environment, run `yarn start`.
```

Comment:
The contributor documentation should not be in the `README`, same issue with the `fast-components` `README`. This should be for developers looking to install the package and use the exports.

Reply:
Sure, this was a complete copy/paste, I'll update prior to merging.